p2p/configuration.go

```diff
@@ -163,9 +172,12 @@ func DefaultP2PConfiguration() (c Configuration) {
 	c.ProtocolVersion = 10
 	c.ProtocolVersionMinimum = 9
 
-	c.ChannelCapacity = 50
+	c.ChannelCapacity = 1000
 
 	c.EnablePrometheus = true
+	c.PeerResend = true
```

Comment:
It sounds odd to say PeerResend is true to signify that we do **not** resend messages.
PeerNoResend? PeerMsgDedupe?

Reply:
Changed it to "PeerResendFilter" which makes more sense in context